components/bio/bio.scss

```diff
@@ -13,6 +13,12 @@
     }
   }
 
+  @media screen and (max-width: 500px) {
```

Comment:
For this codebase we want to use "mobile first" approach so instead of using `max-width`, we want `min-width` here instead.

Learn more about `mobile first` CSS approach: https://zellwk.com/blog/how-to-write-mobile-first-css/

Reply:
Try to stick with breakpoint variables defined in `/scss/overrides/_variables.scss` instead of using an arbitrary number (e.g., 500px).